app/MMB/src/menu/submenu/FlightServices.js

```diff
@@ -0,0 +1,60 @@
+// @flow
```

Comment:
I though it made sense to add a submenu folder, so we might skip suffixing every component with submenu, agree or you think we should keep current structure?

Reply:
Yes, makes sense. We have to evolve the structure as we go...